docker/test-nightly.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+
+const { execSync } = require('child_process');
+
+// Default to 'main' if no argument is provided
+const swiftVersion = (process.argv[2] || 'main').replace(/\./g, '');
```

Comment:
I'd adapt [test-ci.sh](https://github.com/swiftlang/vscode-swift/pull/1105/files#diff-b2b7bb5d8b2aa0763b5a3e50d5f717357080910cc4f81955ca8fd247f966a2e1) to take another argument that specifies we want to run the nightly tests. Then the npm script can do `docker/test-ci.sh nightly` or something similar and we can avoid duplicating this script.